### PR TITLE
Enhancement: Configure visibility_required fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] ~configures the `visibility_required` fixer~
* [x] runs `vendor/bin/ecs check ./src/ ./tests/ --fix`

### CODE REVIEW

File: easy-coding-standard.yaml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 imports:
     - { resource: '%vendor_dir%/lmc/coding-standard/easy-coding-standard.yaml' }
 services:
+    PhpCsFixer\Fixer\ClassNotation\VisibilityRequiredFixer:
```
Comment (OndraM) [inline easy-coding-standard.yaml]: Nice suggestion! I've added this rule to the parent codestyle - could you now please remove this from the PR? (As it  will became redundant once I tag `lmc/coding-standard`)

**Review by OndraM — CHANGES_REQUESTED** [review]

File: easy-coding-standard.yaml (hunk 1 above)
Comment (localheinz) [reply easy-coding-standard.yaml]: Done!

**Review by OndraM — APPROVED** [review]

**Comment by OndraM** [comment]
Thanks @localheinz !

**Comment by localheinz** [comment]
Thank you, @OndraM!

**Pull request merged**